### PR TITLE
Update Oxide to v2.0.3-beta

### PR DESCRIPTION
**Pull request by Eeems**
See release notes for issues fixed: https://github.com/Eeems/oxide/releases/tag/v2.0.2-beta and https://github.com/Eeems/oxide/releases/tag/v2.0.3-beta

### CODE REVIEW

File: package/oxide/package [hunk 1]
```diff
@@ -3,13 +3,13 @@
 # SPDX-License-Identifier: MIT
 
 pkgnames=(erode fret oxide rot tarnish)
-pkgver=2.0.1~beta-4
+pkgver=2.0.2~beta-1
 timestamp=2020-12-06T20:09Z
```
Comment (raisjn) [inline package/oxide/package]: timestamp ;)

**Review by raisjn — DISMISSED** [review]
lgtm

File: package/oxide/package (hunk 1 above)
Comment (Eeems) [reply package/oxide/package]: But I don't wanna :(

*sigh* I really hate this timestamp portion. Give me a sec.

**Review by matteodelabre — APPROVED** [review]
Looks good to me

**Pull request merged**